pom.xml

```diff
@@ -62,6 +62,13 @@
           </dependency>
         </dependencies>
       </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <forkCount>0.8C</forkCount>
+          <reuseForks>true</reuseForks>
```

Comment:
conversly https://github.com/jenkinsci/plugin-pom/pull/453 here be dragons :)

Reply:
> conversly [jenkinsci/plugin-pom#453](https://github.com/jenkinsci/plugin-pom/pull/453) here be dragons :)

When I enabled `reuseForks` it was specifically because the parent pom had disabled them in jenkinsci/plugin-pom#453 .  I did the investigation to confirm that I could reliably run with `reuseForks` without having the tests be unreliable.

I have seen the dragons and am willing to embrace them in this plugin...